Doc/library/winreg.rst

```diff
@@ -473,7 +473,7 @@ This module offers the following functions:
    *type* is an integer that specifies the type of the data. See
    :ref:`Value Types <value-types>` for the available types.
 
-   *value* is a string that specifies the new value.
+   *value* is a string or integer that specifies the new value.
```

Comment:
It can be not only a string or integer. Acceptable types depend on the *type* argument.

Reply:
Specifically None, str, int, list (of str) or implements the buffer protocol.

Reply:
And it is better to document what types are acceptable for specific value of the *type* argument in the corresponding section `value-types`.